src/features/venuemap/useGetAllVenues.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from 'react-query'
+
+import { Venue } from 'features/venue/types'
+import { fetchVenues } from 'libs/algolia/fetchAlgolia/fetchVenues/fetchVenues'
+import { LocationMode } from 'libs/location/types'
+import { useNetInfoContext } from 'libs/network/NetInfoWrapper'
+import { QueryKeys } from 'libs/queryKeys'
+
+export const useGetAllVenues = () => {
+  const netInfo = useNetInfoContext()
+
+  return useQuery<Venue[]>(
+    [QueryKeys.VENUES],
+    () =>
+      fetchVenues({
+        query: '',
+        buildLocationParameterParams: {
+          userLocation: { latitude: 0, longitude: 0 },
+          selectedLocationMode: LocationMode.AROUND_PLACE,
```

Comment:
[question] : C'est normal que ce soit set to `LocationMode.AROUND_PLACE` ?

Reply:
L'implèm actuelle attend un LocationMode, j'ai hésité avec LocationMode.PLACE, mais j'ai pas de préférence 🤷